build.fsx

```diff
@@ -57,13 +57,17 @@ let pushPackage nupkg =
         return result.ExitCode
     }
 
+let analyzeProject projectPath =
+    $"dotnet fsharp-analyzers --project {projectPath} --analyzers-path ./.analyzerpackages/g-research.fsharp.analyzers/ --verbose"
+
 pipeline "Build" {
     workingDir __SOURCE_DIRECTORY__
     stage "RestoreTools" { run "dotnet tool restore" }
     stage "Clean" {
         run (
             cleanFolders
-                [| "bin"
+                [| ".analyzerpackages/g-research.fsharp.analyzers"
```

Comment:
Old versions of analyzer DLLs that are still on disk might be picked up and cause issues.
We could extend the analyzer path but that would mean we have to maintain the version in two places, so let's remove them.

Reply:
I'm a little less in favour of this. Can't we read the `proj` file and target the correct folder that way? I want this to work during a flight on the aeroplane 😏